src/lib/replication/service/raft_repl_service.h

```diff
@@ -64,6 +64,8 @@ class RaftReplService : public GenericReplService,
     void start() override;
     void stop() override;
 
+    void gc_repl_devs() override;
```

Comment:
Why can't we make it stay private and test code do
`#define private public`.

This way we protect the code from leaking concerns right?